### PR TITLE
Dependency added and updated DisparateImpactRemover import 

### PR DESCRIPTION
**Pull request by EddBr**
<ul>
<li><code>networkx</code> was not a dependency in <code>poetry.lock</code> so have now added this.
In order to do so, minimum python version must be >=3.8 </li>
<li> In tutorials/mitigating_bias_tutorials/mitigating_bias_classification.ipynb importing <code>DisparateImpactRemover</code>  created an error. Now fixed.</li>
</ul>

### CODE REVIEW

File: tutorials/mitigating_bias_tutorials/mitigating_bias_classification.ipynb [hunk 1]
```diff
@@ -977,7 +977,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from holisticai.bias.mitigation import DisparateImpactRemover\n",
+    "from holisticai.bias.mitigation.preprocessing.disparate_impact_remover import (DisparateImpactRemover,)\n",
```
Comment (crismunoz) [inline tutorials/mitigating_bias_tutorials/mitigating_bias_classification.ipynb:980]: is this change necessary? we can import directly from holisticai.bias.mitigation

**Review by crismunoz — COMMENTED** [review]
just check the question! The others are fine :)

File: tutorials/mitigating_bias_tutorials/mitigating_bias_classification.ipynb (hunk 1 above)
Comment (EddBr) [reply tutorials/mitigating_bias_tutorials/mitigating_bias_classification.ipynb:980]: You're right :nerd_face: ! This is unnecessary.

For some reason it didn't work when I was testing it before.

**Review by crismunoz — APPROVED** [review]
